data-client/src/main/java/org/wikipedia/login/LoginClient.java

```diff
@@ -33,7 +33,7 @@
 public class LoginClient {
     @Nullable private Call<MwQueryResponse> tokenCall;
     @Nullable private Call<LoginResponse> loginCall;
-
+    @NonNull private String userLang;
```

Comment:
Please add some javadoc to this member, explain why it is needed.

Reply:
Also, could you call it `userLanguage`? In Java we prefer full words.

Reply:
Done!! Sorry for 2 different commits, I messed up my first rebase.